### DISCUSSION TITLE
Notebooks für Städteseiten?

### DISCUSSION THREAD

**Discussion by lhm** [codeformuenster/klimawatch#122] [Ideas]
Beim Erstellen der Seite für Leipzig habe ich festgestellt, das es nützlich wäre, etwas mehr Flexibilität bei der Darstellung zu haben. Die Ziele in Leipzig sind z.B. in "Pro-Kopf" Emissionen formuliert, und es wäre daher eigentlich ganz schön diese auch in dieser Form zu visualisieren. Ich könnte mir vorstellen, das es noch mehr Stadt-spezifische Sonderlocken gibt. Im Notebook Format könnte man diese etwas besser erläutern. In Hugo könnte man die z.B. mit [nb2hugo](https://github.com/vlunot/nb2hugo) integrieren. 

Der Trade-off wäre, das die Städteseiten dann eventuell etwas weniger standardisiert sind, und ich weiss nicht wie wichtig euch das ist. Was meint ihr?

**Comment by silberzwiebel** [comment]
Danke für die Idee und PR (und sorry für späte Rückmeldung, irgendwie verschluckt momentan github e-Mails an mich)!

Ich selber habe erst gestern mein allererstes python-notebook überhaupt erstellt (und fand es, im Vergleich zu Textdateien, erstmal relativ kompliziert) ;) 

Bin also recht neu in der python-Welt unterwegs. Hoffe, dran zu denken, dass bei unserem heutigen Treffen zu diskutieren.

Kurze Nachfrage aber: Den Text einer Stadtseite kannst Du ja mit der Markdown-Datei ändern (und auch viel hinzufügen falls nötig). Die Erstellung des Plots würde ich aus dem Bauch heraus gesagt, lieber in nur einer Datei lassen, da sonst mehr zu maintainen ist. Wo genau läge der Vorteil eines Notebooks?

Zu Prop-Kopf-Emissionen: Das ist ohnehin eine mögliche interessante Art und Weise für noch mehr Visualisierungen ...

**Comment by lhm** [comment]
@silberzwiebel Den Vorteil den ich sehen würde, wäre das es möglicherweise einfacher wäre, angepasste oder erweiterte Visualisierungen zu integrieren, ohne das es gleich für alle Städte sein muss. Ist vielleicht einfacher an einem Beispiel zu erklären, ich werde das mal ausprobieren und sehen wie weit ich komme - bin nämlich auch neu bei Python & Notebooks.
